app/flame_chart/profile_model.ts

```diff
@@ -47,6 +47,11 @@ function eventComparator(a: TraceEvent, b: TraceEvent) {
   return durationDiff;
 }
 
+function threadNameComparator(a: ThreadTimeline, b: ThreadTimeline) {
+  // Group by thread name.
```

Comment:
This comment feels a little weird because this is actually sorting by thread name